.github/workflows/telemetry-check.yml

```diff
@@ -0,0 +1,48 @@
+name: Telemetry
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+
+      - name: Build Keploy
+        run: |
+          go build -v ./...
+
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          java-version: "11"
+          distribution: "temurin"
+          cache: maven
+
+      - name: Setup java Sample project
+        run: |
+          git clone https://github.com/charankamarapu/Sample-Redis_Keploy.git &&
```

Comment:
This is dependent on your repo here. I think we should use application from keploy/samples-java.

Reply:
Added samples-java as a dummy user